ui/src/containers/appView/AppPageContainer.js

```diff
@@ -42,6 +47,45 @@ export class AppPageContainer extends React.Component {
     });
   }
 
+  componentWillReceiveProps (nextProps) {
+    if (nextProps.saveAppFailed && nextProps.saveAppFailed !== this.props.saveAppFailed) {
+      this.showAlert('Save failed', 'danger');
+    }
+
+    if (nextProps.saveAppSuccess && nextProps.saveAppSuccess !== this.props.saveAppSuccess) {
+      this.showAlert('Save successful', 'success');
+    }
+  }
+
+  /**
+   * Show an Alert
+   *
+   * @param {string} title - The alert title
+   * @param {*} variant - one of danger, success, warning, info
+   * @param {*} [description] - displays an optional Alert body
+   * @param {number} [timeout] - optionally set a duration of time before the alert disappears
+   */
+  showAlert (title, variant, description = null, timeout = 5000) {
```

Comment:
`timeout` is no longer required here.

Reply:
My bad, never pushed changes with it gone.